.github/workflows/test-e2e-cli.yml

```diff
@@ -16,6 +16,8 @@ env:
       {"host":"connection.keboola.com","project":8764,"stagingStorage":"s3","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_8764_TOKEN }}"},
       {"host":"connection.keboola.com","project":8765,"stagingStorage":"s3","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_8765_TOKEN }}"},
       {"host":"connection.keboola.com","project":8766,"stagingStorage":"s3","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_8766_TOKEN }}"},
+      {"host":"connection.europe-west3.gcp.keboola.com","project":34,"stagingStorage":"gcs","backend":"bigquery","token":"${{ secrets.TEST_KBC_PROJECT_34_TOKEN }}"},
+      {"host":"connection.europe-west3.gcp.keboola.com","project":29,"stagingStorage":"gcs","backend":"bigquery","token":"${{ secrets.TEST_KBC_PROJECT_29_TOKEN }}"},
       {"host":"connection.north-europe.azure.keboola.com","project":10890,"stagingStorage":"abs","backend":"snowflake","token":"${{ secrets.TEST_KBC_PROJECT_10890_TOKEN }}"},
```

Comment:
@hosekpeter 

- You cannot add same projects to the different test jobs that run in parallel.
  - See, each test job has its own test pojects, see different IDs.
- Please, don't use your DEV project `29`, in the CI pipeline, create a new projects.
- Please use `Keboola Internal` organization, did you get an invitation?
- Please, use an clear name, for example `[ci/go] BigQuery Project 1`, `[ci/go] BigQuery Project 2`.

I think I already explained some of this to you during the onboarding. Sorry if that wasn't enough.
​